src/remote/utils.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const Qs = require('qs')
+const mapValues = require('lodash.mapvalues')
+
+exports.getResponse = (res, cb) => {
+  let data = ''
+  res.on('data', function (buf) {
+    data += buf.toString()
+  })
+
+  res.on('end', () => {
+    const response = JSON.parse(data)
```

Comment:
I think that an HTTP client should do this for you — there shouldn't be the need to buffer and parse it yourself..